### PR TITLE
Bug 1141294 - Move BrowserWindow and sub UI classes into their own sub folder

### PR DESCRIPTION
**Pull request by gbrmachado**

### CODE REVIEW

**Comment by gbrmachado** [comment]
@whimboo 
I've moved both tabbar and toolbars into the new folder. I'm getting one error though.
https://travis-ci.org/mozilla/firefox-ui-tests/jobs/82273145#L1129-L1139
Is it related with my change?

**Comment by whimboo** [comment]
No, it shouldn't be related. Looks like a temporary networking issue. I have retriggered the Travis build. Lets see what the results tell us.

File: firefox_puppeteer/ui/browser/tabbar.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 import firefox_puppeteer.errors as errors
 
-from .. import DOMElement
-from ..api.security import Security
-from ..base import UIBaseLib
+from ... import DOMElement
+from ...api.security import Security
+from ...base import UIBaseLib
```
Comment (whimboo) [inline firefox_puppeteer/ui/browser/tabbar.py]: Maybe we should start to use absolute imports. The multiple dots will be hard to follow. I suggest `from firefox_puppeteer.api.security import Security`. Similar for the other two. With that we also only need a single block because it can be combined with the errors import.

File: firefox_puppeteer/ui/browser/toolbars.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 from marionette_driver import By, keys, Wait
 
-from ..base import UIBaseLib
+from ...base import UIBaseLib
```
Comment (whimboo) [inline firefox_puppeteer/ui/browser/toolbars.py]: Same for the import here.

**Comment by whimboo** [comment]
Please also add the copyright block into the new **init**.py.

File: firefox_puppeteer/ui/windows.py [hunk 3]
```diff
@@ -475,7 +475,7 @@ def navbar(self):
         self.switch_to()
 
         if not self._navbar:
-            from .toolbars import NavBar
+            from browser.toolbars import NavBar
```
Comment (whimboo) [inline firefox_puppeteer/ui/windows.py]: Local (relative) imports should start with a dot.

File: firefox_puppeteer/ui/windows.py [hunk 4]
```diff
@@ -491,7 +491,7 @@ def tabbar(self):
         self.switch_to()
 
         if not self._tabbar:
-            from .tabbar import TabBar
+            from browser.tabbar import TabBar
```
Comment (whimboo) [inline firefox_puppeteer/ui/windows.py]: Same as above.

**Comment by whimboo** [comment]
So the last Travis run succeeded as expected. Beside the nits above your changes look fine!

**Comment by whimboo** [comment]
Looks like the removal of the master branch closed this PR. Please create a new one against mozilla-central. Thanks.
